assets/templates/aws.billing/schema-b/configs.yml

```diff
@@ -0,0 +1,82 @@
+- name: cloud.region
+  enum: ["us-east-1", "us-east-2", "us-west-1", "us-west-2", "ap-south-1", "ap-northeast-3", "ap-northeast-2", "ap-southeast-1", "ap-southeast-2", "ap-northeast-1", "ca-central-1", "eu-central-1", "eu-west-1", "eu-west-2", "eu-west-3", "eu-north-1", "sa-east-1", "af-south-1", "ap-east-1", "ap-south-2", "ap-southeast-3", "eu-south-2", "eu-central-2", "me-south-1", "me-central-1"]
```

Comment:
please. see [this comment](https://github.com/elastic/elastic-integration-corpus-generator-tool/pull/54/files#diff-a5be5ab16f8877b19cdd43994617c91e1459fb8cbeba13bee9ae7cc48518ff67R6-R10) about how to let match the consistency of data between events with cardinality

not sure it applies in for `aws.billing`: but I'd see at list the case where an `aws.billing.group_by.AZ` can be in the "wrong" region (I solved with [this](https://github.com/elastic/elastic-integration-corpus-generator-tool/pull/54/files#diff-fe191b89521118fbf3b0410066ed7014ce38341633d1c8e12e060a6372c1705cR5-R52))

btw, it would be useful to have some macro for that kind of stuff